docs/running-locally.md

```diff
@@ -40,7 +40,7 @@ docker compose --env-file $MITHRIL_CONFIG up -d
 
 To verify the cardano node is actually running you can run:
 ```shell
-cardano-cli query tip --testnet-magic 2 --socket-path "$(docker volume inspect cardanow_cardano-node-ipc | jq -r '.[0]["Mountpoint"]')/node.socket"
+docker compose run cardano-node cli "query tip --testnet-magic 2 --socket-path /ipc/node.socket"
```

Comment:
👍 
In the spirit of abstracting away all the network config to just using the NETWORK env var, we should eventually add some code to figure out the correct testnet-magic based on the env var, but this can definitely wait for later